stl/inc/algorithm

```diff
@@ -1364,6 +1364,54 @@ namespace ranges {
 
     inline constexpr _None_of_fn none_of{_Not_quite_object::_Construct_tag{}};
 
+#if _HAS_CXX23
+    class _Contains_fn : private _Not_quite_object {
+    public:
+        using _Not_quite_object::_Not_quite_object;
+
+        template <input_iterator _It, sentinel_for<_It> _Se, class _Ty, class _Pj = identity>
+            requires indirect_binary_predicate < ranges::equal_to, projected<_It, _Pj>,
+        const _Ty* > //
+            _NODISCARD constexpr bool operator()(_It _First, _Se _Last, const _Ty& _Val, _Pj _Proj = {}) const {
+            return _RANGES find(_STD move(_First), _Last, _Val, _Pass_fn(_Proj)) != _Last;
+        }
+        template <input_range _Rng, class _Ty, class _Pj = identity>
+            requires indirect_binary_predicate < ranges::equal_to, projected<iterator_t<_Rng>, _Pj>,
+        const _Ty* > //
+            _NODISCARD constexpr bool operator()(_Rng&& _Range, const _Ty& _Val, _Pj _Proj = {}) const {
+            return (*this)(_Ubegin(_Range), _Uend(_Range), _Val, _Pass_fn(_Proj));

```

Comment:
We conventionally avoid this additional function call, so I would directly call.
```suggestion
    _RANGES find(_STD forward<_Rng>(_Range), _Val, _Pass_fn(_Proj)) != _Last;
```
Note that calling the iterator version would involve unnecessary valid_range checks

Reply:
I went an extra step here and called `_Find_unchecked` directly since the range flavor of `find` does nothing special.